### PR TITLE
lib/model: little cleanups in model.go

### PR DESCRIPTION
**Pull request by lkwg82**
### Purpose

boyscout rule ... leave it better as you found it

### Testing

ran

### CODE REVIEW

File: lib/model/model.go [hunk 1]
```diff
@@ -675,19 +675,20 @@ func (m *Model) Index(deviceID protocol.DeviceID, folder string, fs []protocol.F
 
 	m.fmut.RLock()
 	files, ok := m.folderFiles[folder]
-	runner := m.folderRunners[folder]
```
Comment (calmh) [inline lib/model/model.go]: I wouldn't make this change. Consider that each RLock/RUnlock cycle is fairly expensive, while the map lookups and the `if !ok` are essentially free.

File: lib/model/model.go [hunk 2]
```diff
@@ -1407,9 +1406,9 @@ func (m *Model) ResumeDevice(device protocol.DeviceID) {
 
 func (m *Model) IsPaused(device protocol.DeviceID) bool {
 	m.pmut.Lock()
-	paused := m.devicePaused[device]
-	m.pmut.Unlock()
```
Comment (calmh) [inline lib/model/model.go]: We sometimes do this for very short methods, as `defer` has a certain cost and there is no great risk of forgetting the unlock in a one line method with a single point of return. In this case it's not a performance critical method though so either defer or not defer is fine, and the defer case is a teensy bit cleaner so 👍

File: lib/model/model.go (hunk 1 above)
Comment (lkwg82) [inline lib/model/model.go]: ok

**Comment by calmh** [comment]
@st-jenkins test this please?

**Comment by calmh** [comment]
@st-review merge it

lib/model: Minor cleanups

**Comment by st-review** [comment]
@calmh: Build status is `pending`. I'll wait until it goes green and then merge!

**Comment by st-review** [comment]
@calmh: Build status is `failure` -- refusing to merge.

**Comment by lkwg82** [comment]
@calmh this build stucks

**Comment by AudriusButkevicius** [comment]
@st-jenkins test this please

**Comment by AudriusButkevicius** [comment]
```
A thread (Thread-248/3167) died unexpectedly due to an uncaught exception, this may leave your Jenkins in a bad way and is usually indicative of a bug in the code.
java.lang.Error: java.io.FileNotFoundException: {"message":"Not Found","documentation_url":"https://developer.github.com/v3"}
	at org.kohsuke.github.Requester$PagingIterator.fetch(Requester.java:433)
	at org.kohsuke.github.Requester$PagingIterator.hasNext(Requester.java:400)
	at org.kohsuke.github.PagedIterator.fetch(PagedIterator.java:44)
	at org.kohsuke.github.PagedIterator.hasNext(PagedIterator.java:32)
	at org.jenkinsci.plugins.ghprb.GhprbPullRequest.tryBuild(GhprbPullRequest.java:365)
	at org.jenkinsci.plugins.ghprb.GhprbPullRequest.check(GhprbPullRequest.java:214)
	at org.jenkinsci.plugins.ghprb.GhprbRepository.onIssueCommentHook(GhprbRepository.java:346)
	at org.jenkinsci.plugins.ghprb.GhprbTrigger.handleComment(GhprbTrigger.java:618)
	at org.jenkinsci.plugins.ghprb.GhprbRootAction$1.run(GhprbRootAction.java:233)
Caused by: java.io.FileNotFoundException: {"message":"Not Found","documentation_url":"https://developer.github.com/v3"}
	at org.kohsuke.github.Requester.handleApiError(Requester.java:592)
	at org.kohsuke.github.Requester$PagingIterator.fetch(Requester.java:429)
	... 8 more
Caused by: java.io.FileNotFoundException: https://api.github.com/repos/syncthing/syncthing/issues/3765/commits
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
	at sun.net.www.protocol.http.HttpURLConnection$7.run(HttpURLConnection.java:1725)
	at sun.net.www.protocol.http.HttpURLConnection$7.run(HttpURLConnection.java:1723)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.net.www.protocol.http.HttpURLConnection.getChainedException(HttpURLConnection.java:1721)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1292)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:254)
	at org.kohsuke.github.Requester.parse(Requester.java:524)
	at org.kohsuke.github.Requester.access$200(Requester.java:68)
	at org.kohsuke.github.Requester$PagingIterator.fetch(Requester.java:424)
	... 8 more
Caused by: java.io.FileNotFoundException: https://api.github.com/repos/syncthing/syncthing/issues/3765/commits
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1674)
	at java.net.HttpURLConnection.getResponseCode(HttpURLConnection.java:468)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:338)
	at org.kohsuke.github.Requester.parse(Requester.java:514)
	... 10 more
```

:(

**Comment by AudriusButkevicius** [comment]
https://github.com/jenkinsci/ghprb-plugin/issues/441

**Comment by AudriusButkevicius** [comment]
@st-jenkins test this please

**Comment by AudriusButkevicius** [comment]
@st-jenkins test this please

**Comment by AudriusButkevicius** [comment]
Btw:

```
--- FAIL: TestGlobalDirectoryTree (0.01s)
panic: sync: RUnlock of unlocked RWMutex [recovered]
	panic: sync: RUnlock of unlocked RWMutex
```

**Comment by AudriusButkevicius** [comment]
@st-jenkins test this please

**Comment by AudriusButkevicius** [comment]
@st-jenkins test this please

**Comment by AudriusButkevicius** [comment]
yay, I fixed jenkins

**Comment by calmh** [comment]
@st-review merge it

lib/model: Minor cleanups

**Comment by st-review** [comment]
:ok_hand: Merged as 8ce9b026e9401d9aa1201fb40bebb350d1024b23. Thanks, @lkwg82!

**Pull request closed without merging**